KP4-Topic3/src/com/aguirre/tests/RomanNumberConverterTest.java

```diff
@@ -0,0 +1,127 @@
+package com.aguirre.tests;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import com.aguirre.classes.RomanNumberConverter;
+
+public class RomanNumberConverterTest {
+
+	@Test
+	public void oneLetter() {

```

Comment:
It's a good practice to try to explain in the name of the testcase exactly what are you testing, for example in this one you could use testRomanToIntWithOneLetter